.github/workflows/docker-build.yml

```
@@ -0,0 +1,93 @@
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)